reactor-netty-core/src/main/java/reactor/netty/resources/PooledConnectionProvider.java

```diff
@@ -105,6 +108,7 @@ protected PooledConnectionProvider(Builder builder) {
 		this.inactivePoolDisposeInterval = builder.inactivePoolDisposeInterval;
 		this.poolInactivity = builder.poolInactivity;
 		this.disposeTimeout = builder.disposeTimeout;
+		this.expectedConnectionPools = builder.expectedConnectionPools;
```

Comment:
IMO it is better to call it maximum connection pools. Wdyt?

```suggestion
		this.maxConnectionPools = builder.maxConnectionPools;
```